pkg/informer/informer.go

```diff
@@ -156,13 +156,12 @@ func NewDynamicDiscoverySharedInformerFactory(
 		}
 	}
 
+	// When CRDs are added or deleted, send a notification that we might need to add/remove informers. We don't
+	// need to handle update events.
```

Comment:
technically there is a time between creation and being Established. Does this mean we ignore that here? (we can, am not against it. But we should be clear in the go doc).

Reply:
still valid

Reply:
How about if we only send notifications for CRDs that are Established & for deletes?